ldm/invoke/args.py

```diff
@@ -547,6 +547,7 @@ def _create_arg_parser(self):
         )
         render_group.add_argument(
```

Comment:
(this reference here)

Reply:
Could you explain what you mean here?

Reply:
i was referring to the previous comment "either embedding_path should be put back or the reference to it below should be deleted"

Reply:
as-is the --embedding_path arg will still be parsed and also reported by -h, but the implementation is ignoring it

Reply:
In latest version the directory or file pointed to by `--embedding_path` will be scanned for embedding files and loaded. Default is `~/invokeai/embeddings`. Embedding loading can be disabled using `--no-embedding`